### PR TITLE
Musixmatch support

### PR DESCRIPTION
**Pull request by intika**
add support for the devil musixmatch

### CODE REVIEW

**Comment by Vincent43** [comment]
There are a lot of commented out options. Can you test working ones and delete the rest?

**Comment by intika** [comment]
hi, 

only 
protocol unix,inet,inet6,netlink
is needed 
thanks

**Comment by Fred-Barclay** [comment]
Hi @intika 
Can you uncomment the following lines and see if musixmatch breaks? If so, try commenting each one individually until you find the line (or lines) that break musixmatch.
```
caps.drop all
netfilter
no3d
nodvd
nogroups
nonewprivs
noroot
nogroups 
nosound
notv
novideo
protocol unix,inet,inet6,netlink
seccomp
shell none
disable-mnt
private-dev
private-tmp
memory-deny-write-execute
noexec ${HOME}
noexec /tmp
```
Thanks!

File: etc/musixmatch.profile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Firejail profile for default
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/default.local
```
Comment (SkewedZeppelin) [inline etc/musixmatch.profile]: s/default/musixmatch/

File: etc/musixmatch.profile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
```
Comment (SkewedZeppelin) [inline etc/musixmatch.profile]: remove this comment

File: etc/musixmatch.profile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# depending on your usage, you can enable some of the commands below:
+
+include /etc/firejail/disable-common.inc
+# include /etc/firejail/disable-devel.inc
```
Comment (SkewedZeppelin) [inline etc/musixmatch.profile]: Enable disable-devel?

File: etc/musixmatch.profile [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+noroot
+nogroups 
+# nosound
+# notv
+# novideo
+protocol unix,inet,inet6,netlink
+seccomp
+# shell none
+
+# disable-mnt
```
Comment (SkewedZeppelin) [inline etc/musixmatch.profile]: remove private, private-bin, private-etc, private-lib?

File: etc/musixmatch.profile [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+# nogroups
+nonewprivs
+noroot
+nogroups 
+# nosound
+# notv
+# novideo
+protocol unix,inet,inet6,netlink
+seccomp
+# shell none
```
Comment (SkewedZeppelin) [inline etc/musixmatch.profile]: enable no3d, nodvd, nogroups, notv, novideo, shell none?

**Comment by intika** [comment]
@Fred-Barclay @Vincent43 @SkewedZeppelin 
Just made a clean profile i will pull request once again :)
Thanks

**Comment by intika** [comment]
new pullrequest https://github.com/netblue30/firejail/pull/1827

**Pull request closed without merging**